### PR TITLE
Fix minor build problems (take 2)

### PR DESCRIPTION
**Pull request by Loquacity**

### CODE REVIEW

File: modules/installation/pages/install-ibmz.adoc [hunk 1]
```diff
@@ -87,12 +87,12 @@ The hostname should contain less than eight characters.
 
 == Installing {productname} on {ibmz}
 
-This section covers the installation of {productname}{productnumber} as an extension to {sles}{nbsp}{sles-version}{sp-version}.
+This section covers the installation of {productname}{productnumber} as an extension to {sls}{nbsp}{sles-version}{sp-version}.
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]:
```suggestion
This section covers the installation of {productname}{nbsp}{productnumber} as an extension to {sls}{nbsp}{sles-version}{nbsp}{sp-version}.
```

File: modules/installation/pages/install-ibmz.adoc [hunk 2]
```diff
@@ -87,12 +87,12 @@ The hostname should contain less than eight characters.
...
 
 For more information on deploying SLES {sles-version}{sp-version}
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]:
```suggestion
```

File: modules/installation/pages/install-ibmz.adoc [hunk 3]
```diff
@@ -87,12 +87,12 @@ The hostname should contain less than eight characters.
...
 on your hardware, see https://www.suse.com/documentation/sles-15/book_sle_deployment/data/cha_zseries.html.
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]:
```suggestion
For more information on deploying SLES {sles-version}{nbsp}{sp-version} on your hardware, see https://www.suse.com/documentation/sles-15/book_sle_deployment/data/cha_zseries.html.
```

File: modules/installation/pages/install-ibmz.adoc [hunk 4]
```diff
@@ -87,12 +87,12 @@ The hostname should contain less than eight characters.
...
 
-. Install {sles}{nbsp}{sles-version}{sp-version} from the installation media, and select {productname} as an extension.
+. Install {sls}{nbsp}{sles-version}{sp-version} from the installation media, and select {productname} as an extension.
```
Comment (keichwa) [inline modules/installation/pages/install-ibmz.adoc]:
```suggestion
. Install {sls}{nbsp}{sles-version}{nbsp}{sp-version} from the installation media, and select {productname} as an extension.
```

**Comment by Loquacity** [comment]
> Looks fine, was Z Systems redefined to IBM Z?

Sure was. Check PR#435

**Pull request merged**